draft-ietf-httpbis-connect-tcp.md

```diff
@@ -230,6 +232,14 @@ IF APPROVED, IANA is requested to add the following entry to the "MASQUE URI Suf
 | Path Segment | Description  | Reference       |
 | tcp          | TCP Proxying | (This document) |
 
+## New Capsule Type {#data-capsule}
+
+IF APPROVED, IANA is requested to add the following entry to the "HTTP Capsule Types" registry:
+
+| ----- | ------------ | --------- | ------------------------------------- | ----------------- | ------- |
+| Value | Capsule Type | Status    | Reference                             | Change Controller | Contact |
+| (TBD) | DATA         | permanent | (This document), {{capsule-protocol}} | IETF              | HTTPBIS |
```

Comment:
Can you add a randomly generated 4-byte capsule type so we can test interop? (make sure to avoid grease values)

Reply:
OK, done.  I've also added a note defining "connect-tcp-05" for interop.